amr-wind/wind_energy/actuator/FLLC.cpp

```diff
@@ -49,6 +112,8 @@ void FLLCParse(const utils::ActParser& pp, FLLCData& data)
     std::string typeString = "variable_chord";
     pp.query("fllc_type", typeString);
     data.correction_type = FLLCTypeMap.at(typeString);
+    pp.query("fllc_nonuniform", data.nonuniform);
+    pp.query("fllc_eps_dr", data.eps_dr);
```

Comment:
A bit nit picky,but this input variable does not match with the others in terms of abbreviation.  For users it is nice to keep the same level and types of abbreviations so `fllc_epsilon_dr`. But I am also not really a fan of this name either.  This is a ratio right?  I have to keep looking in the code to see how it is used to understand what it is.  I think a longer, more descriptive name would be preferred.

Reply:
Yes, I was trying to leave the same format for user inputs so that they know that this is a parameter for the fllc. 
`eps_dr` is the ratio `epsilon/dr` that is used to generate the non-uniform spacing. 
Would `fllc_epsilon_dr_ratio` be a better name?

Reply:
Yeah I like that better